students/8/main.js

```diff
@@ -34,6 +35,21 @@ const ex6 = async () => {
 // Your functions here...
 //
 
+// Ex 2
+
+const countCharsInBody = async (userId) => {
+    try {
+        let result = await axios.get(`https://jsonplaceholder.typicode.com/posts/${userId}`);
+        let chars = Object.keys(result.data.body).length
+        return chars;
```

Comment:
So, something to think about... when axios throws an error, what will `countCharsInBody` in body return?

Reply:
It'll throw a lot of crud instead.